Source/WebKit/UIProcess/Inspector/mac/WKInspectorViewController.mm

```diff
@@ -57,6 +57,9 @@
 
 static NSString * const WKInspectorResourceScheme = @"inspector-resource";
 
+static NSString * const safeAreaInsetsKey = @"safeAreaInsets";
+static void* const webViewKvoSafeAreaInsetsKey = (void*)&webViewKvoSafeAreaInsetsKey;
```

Comment:
Is this really a "key" or should this be called a "context" to match its usage below?

Reply:
Also nit: KVO is an acronym so should be all-caps in this case (i.e. `webViewKVOSafeAreaInsetsContext`)